project/Dependencies.scala

```diff
@@ -104,6 +103,13 @@ object Dependencies {
     ExclusionRule("org.apache.hadoop")
   ) excludeAll(parquetList:_*)
 
+  def sparkCSV: Seq[ModuleID] = {
+    import scala.math.Ordering.Implicits._
+    if (sparkVersionTuple >= (1, 3, 0)) {
```

Comment:
actually you put that in the post-df thingy which is after 1.3.0 so we're safe.

At some point, we'll have to stop supporting these stoneage version :-D